app/models/github_integration.rb

```diff
@@ -6,27 +6,27 @@ class GithubIntegration < ApplicationRecord
   # Reasonably if a user wants to submit code, it should be among
   # the most recent repos updated...
   def repositories
-    if not self.access_token
+    if !access_token
       return nil
     end
 
-    client = Octokit::Client.new(:access_token => access_token)
+    client = Octokit::Client.new(access_token: access_token)
 
     begin
-    repos = client.repos({},
-                         query: {
-                           sort: "pushed",
-                           per_page: 30,
-                         })
-    repos.map { |repo|
-      { repo_name: repo[:full_name],
-        clone_url: repo[:clone_url],
-        default_branch: repo[:default_branch] }
-    }
+      repos = client.repos({},
+                           query: {
+                             sort: "pushed",
+                             per_page: 30,
+                           })
+      repos.map { |repo|
+        { repo_name: repo[:full_name],
+          clone_url: repo[:clone_url],
+          default_branch: repo[:default_branch] }
+      }
     rescue Exception => e
```

Comment:
Do we need to catch all exception here? or just standarderror

Reply:
Good catch, just StandardError. We don't want to deal with stuff raised internally by Ruby